src/model/dataset.py

```diff
@@ -128,17 +118,13 @@ def __getitem__(self, idx: int):
             piece_color: one hot encoding of the piece color
 
         """
-        image_labels_pair = self.image_path_labels_pairs[idx]
-        image = image_labels_pair['image']
-        image_labels = image_labels_pair['labels']
-
-        piece_type = image_labels['piece_type']
-        piece_color = image_labels['piece_color']
-        is_piece = image_labels['is_piece']
+        image_label_pair = self.image_path_labels_pairs[idx]
+        image = image_label_pair['image']
+        image_label = image_label_pair['label']
 
```

Comment:
Do the transform inside the creation of the dataset and not in the `__getitem__` method